### ISSUE TITLE
Dotnet8 support?

### ISSUE THREAD

**Issue by mphilipps** [CycloneDX/cyclonedx-dotnet#774]
```
You must install or update .NET to run this application.

App: /usr/bin/dotnet-CycloneDX
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '7.0.0' (x64)
.NET location: /usr/share/dotnet

The following frameworks were found:
  8.0.0 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=7.0.0&arch=x64&rid=linux-x64&os=debian.11
```

**Comment by NextFreeUsername** [comment]
I have this issue when building and scanning .Net 6.0. This worked fine with 2.9.0 but now it fails like this.
Cause might be the release of .Net 8.0 yesterday which causes .Net 6.0 to no longer be installed on the build server?

**Comment by robmaas** [comment]
> I have this issue when building and scanning .Net 6.0. This worked fine with 2.9.0 but now it fails like this.
Cause might be the release of .Net 8.0 yesterday which causes .Net 6.0 to no longer be installed on the build server?

`net8` is LTS, so might be the case (6 was LTS, 7 wasn't).
Until CycloneDX supports `net8` You can probably fix that by explicitly installing `net6`, for example by using the `UseDotNet@2` task when using Azure DevOps.

**Comment by yorschor** [comment]
There is already a pull request #773  for this. Lets hope it gets reviewed soon ^^

**Comment by CodeTigerCloud** [comment]
If a mere release of a new .net version broke my pipelines, I would consider making them more robust.

[robmaas solution](https://github.com/CycloneDX/cyclonedx-dotnet/issues/774#issuecomment-1813252946) should help you there, though.

dotnet 6 still has support until November 2024. I can guarantee you that dotnet 8 support will be added before that, maybe even this weekend.

**Comment by CodeTigerCloud** [comment]
@Matiszak added the support. Available in version 2.10.0
